Classes/FrontEnd/DefaultController.php

```diff
@@ -2990,7 +2990,7 @@ protected function createRegistrationPage()
 
         $result = $this->createRegistrationHeading($errorMessage);
         $result .= $registrationForm;
-        $result .= $this->getSubpart('REGISTRATION_BOTTOM');
+        $result .= $this->createRegistrationFooter();
```

Comment:
While this code is modified anyway, please let's change it to a chain of concatenations like this:

```php
return $this->createRegistrationHeading($errorMessage)
  . $registrationForm;
  . $this->createRegistrationFooter();
```

(or all in one line if it fits)